playground/src/PlaygroundView.tsx

```diff
@@ -183,6 +188,39 @@ export class PlaygroundView extends React.Component<IPlaygroundViewProps, IPlayg
     );
   }
 
+  private _renderSelectSample(): React.ReactNode {
+    return (
+      <select
+        className='playground-select-sample'
+        value={this.state.selectSampleValue}
+        onChange={this._selectSample_onChange.bind(this)}>
+
+        <option value='none'>Choose a sample...</option>
+        <option value='basic'>A basic example</option>
```

Comment:
Since it looks like `basic` is selected on page load, consider adding `selected` to this `<option/>` so that it is selected by default. Otherwise its a bit confusing because you load the page with a sample in-place, select "Basic", and its the same content.

>You can include a selected attribute on an <option> element to make it selected by default when the page first loads.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select